microsoft-365/security/office-365-security/remediate-malicious-email-delivered-office-365.md

```diff
@@ -32,6 +32,11 @@ Remediation means taking a prescribed action against a threat. Malicious email s
 
 Admins can take required action on emails, but to get those actions approved, they must have the *Search and Purge* role assigned to them in the **Email & collaboration** permissions in the Microsoft 365 Defender portal. Without the *Search and purge"*role added to one of the role-groups, they won't be able to execute the action.
 
+As email actions create Automated investigations in the backend you need to enable 	"Automated Investigation" by navigating Settings >> 
+Endpoints >> Advanced features >>Automated Investigation ( toggle on) 
+
+
```

Comment:
```suggestion
Because email actions create automated investigations in the backend, you need to enable *Automated Investigation*. To enable *Automated Investigation*, navigate to **Settings** > **Endpoints** > **Advanced features** and turn on the **Automated Investigation** toggle.
```